test/db/formats/coff

```diff
@@ -133,3 +133,16 @@ sym._TIFFGetField 0xb469 [CALL] call sym._TIFFVGetField
             0x0001000d      .dword 0x00000000                          ; RELOC 32 _TIFFFindField
 EOF
 RUN
+
+NAME=patching REL32 amd64
+ARGS=-e io.cache=1
+FILE=bins/coff/coff.obj
+CMDS=<<EOF
+pd 2 @ sym.main+6
+EOF
+EXPECT=<<EOF
+            0x0000006a      4c8d051a0000.  lea r8, [0x0000008b]        ; sym..data ; "Win64 assembly"; RELOC 32 .data @ 0x0000008b + 0x0
```

Comment:
this + 0x0 makes no sense. can we get rid of them?

Reply:
Of course but it requires changes in `core/disasm.c` and isn't related to the current PR.

Reply:
Good, then looks good to merge, can you open a separate PR for this? Thanks!

ill fix the PR title now

Reply:
#17312